install/preinstall_odoo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eo pipefail
+
+ODOO_URL=https://github.com/odoo/odoo/archive/$ODOO_VERSION.tar.gz
+
+curl -o /tmp/odoo.tar.gz -SL $ODOO_URL
+tar xfz /tmp/odoo.tar.gz -C /opt/odoo/
+mv /opt/odoo/odoo-$ODOO_VERSION/ /opt/odoo/src
+pip install -e /opt/odoo/src
+rm -f /tmp/odoo.tar.gz
+rm -rf /opt/odoo/src
```

Comment:
overall looks good to me, just a question about this: am I wrong or this is going to run every time, dev mode included?
How it deals w/ forked/aggregated odoo/src via submodule?
Also, wouldn't be better to make `ODOO_URL` a global var so that we can override it IF needed?

Reply:
In fact it do not really install odoo. It only install it (with pip dev mode) and remove the source. There is not odoo code in the image, you only have the python bin and the link to the source code.
So basically when you are in dev and you start your docker the first time
- the entrypoint will generate the odoo.egg in your odoo directory
- the link in lib and the odoo bin already exist (nothing done here)

Then if you restart your docker, as the odoo.egg already exist and as bin and lib link already exist, it work without using onbuild !

Note it will be also possible to not preinstall odoo but this mean installing it in the Dockerfile of your project. But as odoo code source is huge, building the context take time and I do not want to wait !